todo_app/templates/index.html

```diff
@@ -12,7 +12,15 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {%for items in items%}<!--Used flask notation to loop through the items list-->
+          <li>{{ items.id }} - {{ items.status }} - {{ items.title }}</li><!--Used flask notation to add items into the unordered list-->
+        {%endfor%}
       </ul>
+      <form action="/add" method="post"><!--Added a form to post data to the app-->
+        <label for="item">Add Item</label>
+        <input type="text" id="item" name="item"><br>
```

Comment:
You could add the `required` attribute to the input element to make sure you don't get any empty titles.